spec/presenters/user_presenter_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe UserPresenter do
+  describe 'MODEL_ATTRIBUTES constant' do
+    it 'contains all model fields' do
+      expect(UserPresenter::MODEL_ATTRIBUTES).to eq User.attribute_names.map(&:to_sym)
+    end
+  end
+
+  describe '#page_content' do
+    let(:user) { build(:user) }
+    subject { described_class.new(user).page_context }
+
+    it { is_expected.to be_instance_of Hash }
+  end
```

Comment:
check what exactly this method returns

Reply:
![Screenshot at 2020-03-11 11-21-37](https://user-images.githubusercontent.com/52406903/76401484-9c787c80-638a-11ea-9298-d31966996252.png)
hm, it seems it is hash

Reply:
I mean you need to create an test example which will check returned data.